src/main/java/org/spongepowered/common/mixin/core/data/types/MixinBlockStoneSlabNewEnumType.java

```diff
@@ -47,4 +53,12 @@
     public String slab$getName() {
         return shadow$getUnlocalizedName();
     }
+
+    public Translation slab$getTranslation() {
```

Comment:
@gabizou requested me to use lazy evaluating fields [here](https://github.com/SpongePowered/SpongeCommon/pull/1306#discussion_r111633951). Shall I change it?